app/code/Magento/Developer/Console/Command/TemplateHintsShowCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ConfigResource\ConfigInterface;
+
+class TemplateHintsShowCommand extends Command
+{
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:template-hints:show';
+
+    /**
+     * Success message
+     */
+    const SUCCESS_MESSAGE = "Template hints are ";
+
+    /**
+     * @var \Magento\Framework\App\Config\ScopeConfigInterface
+     */
+    protected $scopeConfig;
+
+    /**
+     * Initialize dependencies.
+     *
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     */
+    public function __construct(
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+    )
+    {
+        parent::__construct();
+        $this->scopeConfig = $scopeConfig;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
`@inheritdocs` without `{}`

Reply:
I've just copied from core app/code/Magento/Developer/Console/Command/TemplateHintsEnableCommand.php and app/code/Magento/Developer/Console/Command/TemplateHintsDisableCommand.php classes.

Reply:
Done!